tests/pootle_app/views.py

```diff
@@ -78,3 +80,37 @@ def test_view_welcome_garbage(client):
         with memusage() as usage:
             client.get(url)
         assert not usage["used"]
+
+
+@pytest.mark.django_db
+def test_view_index_redirect(client, language0, project0, request_users):
+    user = request_users["user"]
+    client.login(
+        username=user.username,
+        password=request_users["password"])
+    response = client.get("/")
+    if user.is_authenticated:
+        assert response.status_code == 302
+        assert response.url == "/projects/"
+    else:
+        assert response.status_code == 200
+
+    # cookie test
+    client.cookies["pootle-language"] = language0.code
+    response = client.get("/")
+    if user.is_authenticated:
+        assert response.status_code == 302
+        assert response.url == "/%s/" % language0.code
+    else:
+        assert response.status_code == 200
+
+    # accept lang test
+    del client.cookies["pootle-language"]
+    es = Language.objects.create(code="es")
+    TranslationProject.objects.create(language=es, project=project0)
+    response = client.get("/", HTTP_ACCEPT_LANGUAGE="es")
```

Comment:
I would add test for `HTTP_ACCEPT_LANGUAGE=*` and `HTTP_ACCEPT_LANGUAGE=es-ES` too.